WebContent/WEB-INF/jsp/events.jsp

```diff
@@ -123,11 +123,25 @@
 		
 //         doSearch(0,time.getTime());
         console.log("newSearch");
-        if(!isValid(eventId.value)) {
-            $set("searchMessage", $get("eventIdLabel") + " - Incorrect input data type");
-        } else if(!isValid(maxResults.value)) {
-            $set("searchMessage", $get("maxResultsLabel") + " - Incorrect input data type");
-        } else {
+        if (!isValid(eventId.value)) {
```

Comment:
It needs to be done in the same way as it was done, as in the problem in the topic, correct the message itself, this solution is unreadable, we have to calculate the condition in many places, it is unreadable and will generate increasingly worse code.